rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```diff
@@ -171,6 +172,80 @@
           "pattern": "^https://ils.rero.ch/api/organisations/.*?$"
         }
       }
+    },
+    "notes": {
+      "title": "Notes",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "title": "Note",
+        "propertiesOrder": [
+          "type",
+          "content"
+        ],
+        "required": [
+          "type",
+          "content"
+        ],
+        "properties": {
+          "type": {
+            "type": "string",
+            "title": "Type",
+            "enum": [
+              "public_note",
```

Comment:
Do I need to have `note` in `public_note`? I already know that it is a note. Maybe only `public`?

Reply:
I need to use a context for note type. 
If I only use `checkout` instead of `checkout_note`, when I want to translate the note_type into UI, the return french string will be "prêt" but we would like to display "note d'avertissement de prêt"

Reply:
according to @iGormilhit, we keep 'xxx_note' as value, we use "xxx note" as label (without underscore)